.github/workflows/main.yml

```diff
@@ -0,0 +1,53 @@
+name: Update README
+
+on:
+  workflow_dispatch:
+  push:
+  schedule:
+    # run at 10am UTC daily
+    - cron: '0 10 * * *'
```

Comment:
Currently runs daily at 10AM UTC, we might wanna change that. How often do we want the action to run?

Reply:
Yeah, I don't think these specs update that often. Maybe once a week alongside the test result updates (i.e. over the weekend) would be sufficient?

Reply:
Sounds good. I've updated it to run at 10 AM UTC every Sunday